pyanaconda/modules/storage/bootloader/efi.py

```diff
@@ -69,8 +69,10 @@ def _add_single_efi_boot_target(self, partition):
         boot_disk = partition.disk
         boot_part_num = str(partition.parted_partition.number)
 
+        create_method = "-C" if self.keep_boot_order else "-c"
```

Comment:
You have to disable pylint for this line as it is done in on line 125.

Reply:
Oh, ouch, yes. I think there is some way to disable it for the whole file? Spamming the disables everywhere does not seem very useful...